compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -43,9 +43,11 @@ import org.w3c.dom.HTMLCanvasElement
 import org.w3c.dom.HTMLStyleElement
 import org.w3c.dom.HTMLTitleElement
 
-internal actual class ComposeWindow(val canvasId: String)  {
-
-    actual constructor(): this(defaultCanvasElementId)
+@OptIn(InternalComposeApi::class)
+private class ComposeWindow(
+    val canvasId: String,
+    val content: @Composable () -> Unit,
```

Comment:
I don't really like `@Composable` as constructor parameter why are we moving it to here?

Reply:
Could you please share what's wrong in your opinion with Composable in a constructor? 

It's a private class in this case.  So users are not affected at all.

Reply:
We can simplify to not use `val` in JS sourceSet.
Just simple argument without property.

Reply:
Because semantically it can be changed during the lifetime of the object. We should not limit this ability during API design (public or internal)

Reply:
I see. 
Should such a need arise, private class can be easily modified.
And Dima has already removed 'val'.

Reply:
My vote here is to revert `setContent` method

Reply:
Composable content was moved to constructor in other targets too. 
My vote is to keep it as is, simply because YAGNI: If/When we need it, it can be changed again.

Reply:
For example, in uikit it was `private lateinit var content: @Composable () -> Unit` and `setContent` method.
Now, lateinit var is gone.
Having a lateinit var is less straighforward than a constructor parameter.

Reply:
I'm OK to change it later, so not blocking this PR

Reply:
Thanks! Let's stay in constructor for now. It is simpler for all platforms.